readme.md

```diff
@@ -1,3 +1,127 @@
-# Pine Icon Libary
+# Pine Icon Library
+
+A simple way to add beautiful icons to your web projects using Kajabi's Pine Design System. These icons work in any modern browser and are easy to customize.
+
+[![npm version](https://badge.fury.io/js/%40pine-ds%2Ficons.svg)](https://badge.fury.io/js/%40pine-ds%2Ficons)
+
+## What's Included
+- Ready-to-use SVG icons that load fast
+- Works in any modern browser
+- Easy to change colors and sizes
+- Works with TypeScript
+- Automatic updates from Figma designs
+- Built for accessibility
+
+## Quick Start
+
+Choose one of these two simple ways to start:
+
+### 1. Direct Browser Use (No Framework Needed)
+
+Add this line to your HTML:
+```html
+<script type="module" src="https://cdn.jsdelivr.net/npm/@pine-ds/icons@9.1.0/dist/pds-icons/pds-icons.esm.js"></script>
+```
+
+Then use icons anywhere in your HTML:
+```html
+<pds-icon name="icon-name"></pds-icon>
+```
+
+### 2. Using with React/npm
+
+Install the package:
+```bash
+npm install @pine-ds/icons
+```
+
+Use in your code:
+```jsx
+<pds-icon name="icon-name"></pds-icon>
+```
```

Comment:
Could this be removed since the next section is "How to use" and has the same example?